bagitobjecttransfer/recordtransfer/forms/transfer_forms.py

```diff
@@ -902,4 +1006,137 @@ class ReviewForm(TransferForm):
     it.
     """
 
+    class Meta:
+        """Meta information for the form."""
+
+        transfer_step = TransferStep.REVIEW
+
     captcha = ReCaptchaField(widget=ReCaptchaV2Invisible, label="hidden")
+
+    @staticmethod
+    def _get_base_fields_data(form: BaseForm) -> dict[str, Any]:
+        """Extract fields data from a form."""
+        return {
+            form.fields[field].label or field: form.cleaned_data.get(field, "")
+            for field in form.fields
+            if form.fields[field].label != "hidden"
+        }
+
+    @staticmethod
+    def _process_file_upload(form: UploadFilesForm, user: User) -> dict[str, Any]:
```

Comment:
To make consistent with `_get_base_fields_data`:

```suggestion
    def _get_file_upload_fields_data(form: UploadFilesForm, user: User) -> dict[str, Any]:
```

Reply:
Addressed by:
https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/329/commits/6b727ae5950d5196799997dba3336eee36f904d7